Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -678,6 +699,66 @@ protected void SetupEventOptions()
             eventOptions = InteractableEvent.GetEventTypes();
         }
 
+        protected string[] GetEventList()
+        {
+            return new string[] { };
+        }
+
+        #endregion Events
+
+        #region PopupUtilities
+
+        /// <summary>
+        /// Look for speech commands in the MRTK Speech Command profile
+        /// </summary>
+        /// <param name="keywords"></param>
+        /// <returns></returns>
+        protected bool TryGetSpeechKeywords(out string[] keywords)
+        {
+            List<string> keys = new List<string>();
+            
+            if (MixedRealityToolkit.Instance && (MixedRealityToolkit.Instance.ActiveProfile.IsInputSystemEnabled ||
+                MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.SpeechCommandsProfile != null ||
```

Comment:
@davidkline-ms  is this the proper / recommended way to get MRTK profiles now?

Reply:
Should I follow this pattern?
``` C#
if (!MixedRealityToolkit.ConfirmInitialized() || !MixedRealityToolkit.Instance.HasActiveProfile)
            {
                descriptionsArray = null;
                return false;
            }

            MixedRealityInputAction[] actions = InputSystem.InputSystemProfile.InputActionsProfile.InputActions;
```

Reply:
I would say yes, though looking to @Railboy and @davidkline-ms for the blessing since they own that system. But the ConfirmInitialized() has been the pattern I've been following.

Reply:
Updated to use similar pattern for getting input actions.